docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -1346,6 +1346,11 @@ Query Parameters:
 -   **memswap** - Total memory (memory + swap), `-1` to disable swap.
 -   **cpushares** - CPU shares (relative weight).
 -   **cpusetcpus** - CPUs in which to allow execution (e.g., `0-3`, `0,1`).
+-   **buildargs** – JSON map of string pairs for build-time variables. Users can
+        set these values at build-time and they are used as environment context
+        for the command(s) run as part of the Dockerfile's `RUN` primitive or
+        for variable expansion in other Dockerfile primitives . Read more about
+        `ARG` primitive  [here](https://docs.docker.com/reference/builder/#arg)
```

Comment:
can you change "primitive(s)" to "instruction(s)" in this paragraph? We usually refer to them as "instructions"

Also, could you change the absolute URL to `/reference/builder/#arg`?

Oh, and a final nit; "Read more about *the* `ARG` ....." ("the" was missing)